mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -193,20 +208,33 @@ class KMeans private (
 
     val initStartTime = System.nanoTime()
 
-    val centers = if (initializationMode == KMeans.RANDOM) {
-      initRandom(data)
-    } else {
-      initKMeansParallel(data)
-    }
+    // Only one run is allowed when initialModel is given
+    val numRuns = if (initialModel.nonEmpty){
+      if (runs >1 ) logWarning("Ignoring runs; one run is allowed when initialModel is given.")
```

Comment:
Please replace your current code with the snippet I wrote above (defining numRuns).  The current code does not follow the style guide.